src/custom/state/orders/reducer.ts

```diff
@@ -35,12 +35,19 @@ export interface OrderObject {
 type OrdersMap = Record<OrderID, OrderObject>
 export type PartialOrdersMap = Partial<OrdersMap>
 
+export enum OrderTypeKeys {
+  PENDING = 'pending',
+  FULFILLED = 'fulfilled',
+  CANCELLED = 'cancelled',
+  EXPIRED = 'expired',
+}
```

Comment:
Enum? Nooooooooooooooo!1!!

Well, you know, still up to you.

Reply:
love me some enums